platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/command/ProfileListCommandTest.java

```diff
@@ -0,0 +1,129 @@
+package org.codice.ddf.admin.application.service.command;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.karaf.bundle.core.BundleService;
+import org.apache.karaf.bundle.core.internal.BundleServiceImpl;
+import org.apache.karaf.features.Dependency;
+import org.apache.karaf.features.Feature;
+import org.apache.karaf.features.FeaturesService;
+import org.apache.karaf.features.internal.service.FeaturesServiceImpl;
+import org.codice.ddf.admin.application.service.ApplicationService;
+import org.codice.ddf.admin.application.service.impl.ApplicationServiceImpl;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class ProfileListCommandTest {
+    private ApplicationService applicationService;
+    private FeaturesService featuresService;
+    private BundleService bundleService;
+
+    private PrintStream console;
+    private ByteArrayOutputStream out;
+    private ProfileListCommand profileListCommand;
+
+    @Rule
+    public TemporaryFolder ddfHome = new TemporaryFolder();
+
+    private Path profilePath;
+
+    @Before
+    public void setup() throws IOException {
+        ddfHome.newFolder("etc", "admin", "profiles");
+        profilePath = Paths.get(ddfHome.getRoot().toString(), "etc", "admin", "profiles");
+        Files.copy(this.getClass().getResourceAsStream("/profiles/devProfile.json"), Paths.get(profilePath.toAbsolutePath().toString(), "devProfile.json"));
+        Files.copy(this.getClass().getResourceAsStream("/profiles/profileWithDuplicates.json"), Paths.get(profilePath.toAbsolutePath().toString(), "profileWithDuplicates.json"));
+        out = new ByteArrayOutputStream();
+        console = new PrintStream(out);
+        profileListCommand = new ProfileListCommand();
+        profileListCommand.console = console;
+        profileListCommand.profilePath = profilePath;
+
+        this.applicationService = mock(ApplicationServiceImpl.class);
+        this.featuresService = mock(FeaturesServiceImpl.class);
+        this.bundleService = mock(BundleServiceImpl.class);
+
+        Feature feature = createMockFeature("standard");
+        List<Dependency> deps = Arrays.asList(createMockDependency("app1"),
+                createMockDependency("app2"), createMockDependency("app3"));
+        when(feature.getDependencies()).thenReturn(deps);
+        when(applicationService.getInstallationProfiles()).thenReturn(Collections.singletonList(feature));
+
+    }
+
+    @Test
+    public void testProfileFilesExist() {
+        profileListCommand.doExecute(applicationService, featuresService, bundleService);
+        assert out.toString().contains("standard");
```

Comment:
❗️ Don't think we want to use the `assert` keyword in unit tests. We should use `assertThat()` instead.